utils.go

```diff
@@ -169,3 +169,10 @@ func NewMysqlPosition(file string, position uint32, err error) (mysql.Position,
 		}, nil
 	}
 }
+
+func CheckDbIsAReplica(db *sql.DB) (bool, error) {
+	row := db.QueryRow("SELECT @@read_only")
```

Comment:
Have we thought about using `SHOW SLAVE STATUS` here instead of just relying on the `@@read_only` variable? It seems it is technically possible to run an async slave without setting this flag as MySQL doesn't force you to set `read_only = ON`. What do you think?

Reply:
Shopify runs a master-master configuration which means masters from different location are slaving off each other (and hence SLAVE STATUS will show the slave thread on both instances). We're tracking making this configurable in https://github.com/Shopify/ghostferry/issues/53